docs/Fallback_fonts.md

```diff
@@ -0,0 +1,18 @@
+# Font fallback #
+
+FPDF allows you to specify a list of fonts to be used if one character is not available on the font currently used. Common scenarios are use of special characters like emojis within your text or mixing different languages.

```

Comment:
👍 Nice documentation, thanks for writing it!

It think this section may be better of in `docs/EmojisSymbolsDingbats.md`,
with maybe also a link about the font fallback mechanism in `docs/Unicode.md` pointing to it

Reply:
Actually I think it should be the other way around.
The explanation on how to do it is not specific to emoji at all, and belongs to "Unicode.md".

In the current phrasing, emoji are simply the only example given, which seems misleading all by itself. Another quite common use case is eg. greek characters in formulas, or pretty much any non-latin script turning up eg. in citations.

Of course it needs to be mentioned in "EmojisSymbolsDingbats.md", but that's only one of many possible use cases.

Reply:
I'm fine with sections being structured the other war around, in the order suggested by @gmischler 😊